products/ckeditor.md

```diff
@@ -0,0 +1,35 @@
+---
+title: CKEditor
+category: framework
+permalink: /ckeditor
+releaseColumn: true
+activeSupportColumn: true
+extendedSupportColumn: true
+releasePolicyLink: https://ckeditor.com/docs/ckeditor5/latest/updating/versioning-policy.html
+iconSlug: ckeditor4
+
+releases:
+-   releaseCycle: "5"
+    eol: false
+    support: true
+    latest: "41.1.0"
+    link: https://github.com/ckeditor/ckeditor5/releases/tag/v41.1.0
+    latestReleaseDate: 2024-02-07
+    extendedSupport: false
+
+-   releaseCycle: "4"
```

Comment:
Can you sort fields by following [the guidelines](https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide) ?